vendor/TraceKit/tracekit.js

```diff
@@ -152,7 +152,11 @@ TraceKit.report = (function reportModuleWrapper() {
         if (lastExceptionStack) {
             TraceKit.computeStackTrace.augmentStackTraceWithInitialElement(lastExceptionStack, url, lineNo, message);
             processLastException();
-        } else if (ex) {
+        } else if (ex && typeof ex === 'object') {
```

Comment:
But you did remind me of one thing, which is that you could do `throw new String('lol')` that would evaluate as an `object`.

Gonna revisit.